summergame.module

```diff
@@ -536,9 +538,8 @@ function summergame_dod_response($text, $pid) {
 
   if ($new_track = $track_lookup[$text]) {
     // check if they've already received points for this track
-    $row = db_fetch_array(db_query("SELECT * FROM sg_ledger WHERE pid = %d " .
-                                   "AND metadata LIKE '%%track:%s%%' AND points > 0",
-                                   $pid, $new_track));
+    $row = db_fetch_array($db->query("SELECT * FROM sg_ledger WHERE pid = $pid " .
```

Comment:
Guessing we don't want to call db_fetch_array() here in addition to changing the db_query()

Reply:
Removed the remaining db_fetch_array.